Sources/PointFree/Admin/AdminMiddleware.swift

```diff
@@ -0,0 +1,66 @@
+import ApplicativeRouter
+import Foundation
+import HttpPipeline
+import Models
+import PointFreeRouter
+import Prelude
+import Tuple
+
+let adminMiddleware =
+  requireAdmin
+    <| _adminMiddleware
```

Comment:
I pulled out all admin related routes/middleware into a new file. this also allows me to `requireAdmin` a single time, which allowed me to remove a bunch of `requireAdmin`'s in other middlewares